integration-testing/examples/mpp-sample/build.gradle.kts

```diff
@@ -50,6 +50,11 @@ kotlin {
     }
 }
 
+// Workaround for KT-71203. Can be removed after https://github.com/Kotlin/kotlinx-atomicfu/issues/431
+atomicfu {
+    transformJs = false
```

Comment:
Should user apply the same change to their projects to use AFU w/ this PR applied without having "clashing `unique_name` property" error?

Reply:
I'm preparing a fix for that rn: I'll set `transformJs` value to false by default

Reply:
But if, for some reason, they have this property explicitly enabled, they will get the cryptic error message anyway?

Reply:
And I'll deprecate this property with a warning

Reply:
Maybe we should also print an explicit warning describing the problem to guide users toward the solution? Personally, I'm struggling to link `KLIB resolver blablabla unique_name blablabla` with `atomicfu.transformJs` :)

As of https://github.com/Kotlin/kotlinx-atomicfu/issues/431, I remember we agreed that JS transformation removal sounds OK, but the final decision was pending (some additional research over potential downsides was required).

Reply:
Regarding #431 it's only related to JS IR, and I would do deprecation (and preliminary checks) of JS IR transformations for the next release.

`transformJs` only disabled/enabled legacy JS transformation and now, being still enabled by default, it just creates a new js directory, causing the problem, that you've mentioned. Removing `transformJs` flag and legacy transformation tasks would solve the problem 👀

Reply:
Got it, thanks.

> Removing transformJs flag and legacy transformation tasks would solve the problem 👀

Sure, but there are a few projects that are still using it.

Reply:
I've prepared a PR removing legacy transformations and disabling/deprecating transformJs flag here: https://github.com/Kotlin/kotlinx-atomicfu/pull/478

With these changes, transformJs flag does not take any effect (specifically does not create an empty is directory which caused the above mentioned error), so for the user, who used transformJs there will just be a flag deprecation warning and the WA for this error will not be needed.